scripts/run.py

```diff
@@ -0,0 +1,176 @@
+        # See https://en.wikipedia.org/wiki/Norwegian_Wikipedia
+        'nb': 'no',
+        # There's only one Portuguese Wikipedia, so go with Portugal's Portuguese translation
+        # See https://en.wikipedia.org/wiki/Portuguese_Wikipedia
+        'pt-PT': 'pt',
+        # Simplified and Traditional Chinese
+        # The supported codes are a relic from old Chinese Wikipedia. Nowadays localisation is done on the fly.
+        # See https://en.wikipedia.org/wiki/Chinese_Wikipedia
+        'zh-Hans': 'zh-cn',
+        'zh-Hant': 'zh-tw'
+    }
+    supported_code_to_filename = {}
+    for file_info in xmltodict.parse(filename_to_content['index.xml'])['udhrs']['udhr']:
+        supported_code = bcp47_code_to_supported_code.get(file_info['@bcp47'], file_info['@bcp47'])
+        if supported_code in _SUPPORTED_LANGUAGES:
+            # Some languages have multiple translations, so we just use the last one (which seems to be more recent)
+            supported_code_to_filename[supported_code] = 'udhr_{}.txt'.format(file_info['@f'])
+    assert len(_SUPPORTED_LANGUAGES) == len(supported_code_to_filename)
+
+    # Write the selected translations to the output file
+    whitespace_pattern = re.compile(r'\s+')
+    with open(out_path, 'w', encoding='utf-8') as out_file:
+        for supported_code, filename in sorted(supported_code_to_filename.items()):
+            # Remove the first 6 lines (English header) and clean up whitespace
+            clean_text = whitespace_pattern.sub(' ', ' '.join(filename_to_content[filename].split('\n')[6:])).strip()
+            out_file.write('{}\t{}\n'.format(supported_code, clean_text))
+
+
+@baker.command
+def generate_wordpress_translations_dataset(out_path=os.path.join(_TEST_RESOURCES_PATH, 'wordpress-translations.tsv'),
+                                            texts_per_language=50):
+    """
+    Download and generate the WordPress interface translations dataset.
+
+    The generated dataset consists of translations for WordPress 4.6.x versions. This command applies minimal processing
+    to create the dataset, including: matching the dataset's language codes with those used by the plugin, unescaping
+    HTML entities, and stripping variable placeholders, HTML tags, and redundant whitespace.
+
+    :param out_path: output path for the dataset file, which will be written in tab-separated format with two
+                     columns: language code and text.
+    :param texts_per_language: number of texts to retain per langauge. The output file will contain up to this number of
+                               texts per language, excluding URL translations and word lists. The longest texts for
+                               each language are retained.
+    """
+    url_template = 'https://translate.wordpress.org/projects/wp/4.6.x/{}/default/export-translations?format=json'
+    requests_session = requests.Session()
+    wp_placeholder_pattern = re.compile(r'(%\d*\$?[sd])|(###[A-Z_]+###)')
```

Comment:
This doesn't completely filter all placeholders, but I think that it's not necessarily a bad thing to leave some Latin characters in the texts, as they also occur in organically-created texts.